types/index.d.ts

```diff
@@ -445,6 +445,26 @@ declare namespace RamdaAdjunct {
         paths(ps: Array<Array<string | number>>, obj: object): any[];
         paths(ps: Array<Array<string | number>>): (obj: object) => any[];
 
+        /**
+         * If the given, non-null object has a value at the given path, returns the value at that path.
+         * Otherwise returns the result of invoking the provided function with the object.
+         */
+        pathOrLazy<T>(
+            defaultValueFn: () => T,
+            path: Array<number | string>,
+            obj: object
+          ): T;
+          pathOrLazy<T>(
```

Comment:
The indentation here looks off. Do you see it too ?

Reply:
yes you were right... I just updated it. good catch! 👍